install/docker-extension/ui/config-overrides.js

```diff
@@ -0,0 +1,22 @@
+const webpack = require('webpack');
+module.exports = function override(config, env) {
```

Comment:
@sudhanshutech why don't we use a webpack plugin for this - https://www.npmjs.com/package/node-polyfill-webpack-plugin

Reply:
This plugin is not needed when we using the react app rewired so we can easily configure the webpack configurations.
And when i gone through there some discussion it not solving few things completely.